### DISCUSSION TITLE
Showing the status of all nodes in SOC

### DISCUSSION THREAD

**Discussion by OzzyKampha** [Security-Onion-Solutions/securityonion#1816] [Ideas] [answered]
It would be nice if the status of all the nodes was showed in the SOC, so it was easier to see if it was any problems with the nodes
in the sensors tab, one group for sensor, search, fleet,heavy, master. if they are present in the deployment

**Comment by dougburks** [comment, marked as answer]
We're starting to work on this in https://github.com/Security-Onion-Solutions/securityonion/issues/2039 and https://github.com/Security-Onion-Solutions/securityonion/issues/2040. Look for these changes in 2.3.20 (https://github.com/Security-Onion-Solutions/securityonion/projects/16).
